### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yasharthratan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
+[![UI ](https://img.shields.io/badge/User%20Interface-Link%20to%20UI-orange?style=flat-square&logo=appveyor)](https://github.com/CodeChefVIT/LeoClub-Website/blob/master/index.html)
 
 ## Features
-- Website is made by using HTML , CSS and Bootstrap.
+- Website is made by using HTML , CSS , Jquery and Bootstrap.
 - Contains information about LeoClub VIT , their previous events and their current board and committe members.
 
 ## Screenshots
-<img src="https://github.com/Aarushi21/LeoClub-Website/tree/master/assets/img/screenshot" alt="Project Screenshots">
+<img src="https://github.com/CodeChefVIT/LeoClub-Website/blob/master/assets/img/screenshot.png" alt="Project Screenshot">
```
Comment (vinitshahdeo) [inline README.md:17]: @yasharthratan Please use markdown syntax.

Comment (vinitshahdeo) [reply README.md:17]: Any specific reason for using `HTML` tags?
Is it the part of the template provided?

Comment (akshatvg) [reply README.md:17]: @vinitshahdeo 

Yes we have used HTML tags in the official template for images (to assign custom dimensions) and paragraph tags to center attribute to center stuff.

Comment (vinitshahdeo) [reply README.md:17]: Cool 👍

I appreciate your work. You guys are doing great. Keep it up:)

Comment (akshatvg) [reply README.md:17]: @vinitshahdeo thank you 😊

**Pull request merged**